cluster/gce/gci/configure-helper.sh

```diff
@@ -2233,7 +2233,12 @@ function start-cloud-controller-manager {
     params+=" --configure-cloud-routes=false"
   fi
   if [[ -n "${FEATURE_GATES:-}" ]]; then
-    params+=" --feature-gates=${FEATURE_GATES}"
+    # remove non-GCP feature gates, since the CCM will early exit
+    # if given a feature gate it doesn't recognize
+    local safe_feature_gates=$(echo "${FEATURE_GATES}" | grep --perl-regexp -o "((${GCP_FEATURE_GATE_FILTER})=(true|false),*)" | tr -d "\n") 
+    if [[ -n "${safe_feature_gates:-}" ]]; then
+      params+=" --feature-gates=${safe_feature_gates}"
```

Comment:
Can we add an else clause with a warning written to the kube-up/init logs that we filtered out feature gate X from being sent to the CCM?

Reply:
Gotcha, fixed!